projects/table/src/lib/interfaces/internal-helpers.ts

```diff
@@ -0,0 +1,5 @@
+// Iben: As seen in https://learn.microsoft.com/en-us/javascript/api/@azure/keyvault-certificates/requireatleastone?view=azure-node-latest
```

Comment:
Misschien moeten we eens zien om die de ts utils package te steken? (niet perse nu, maar op het board zetten als een todo).

Reply:
Ik dacht dat ook, maar ik ga dan eerst eens dubbelchecken of dat niet gewoon al als package bestaat 😅